File: Programming Fundametals - Python май 2022/Homeworks - Labs and Exercises/Functions - Exercises/04. Odd and Even Sum.py
def sum_even_odd_digits(n):
    even = 0
    odd = 0
    nums = []
    for digits in n:
        nums.append(digits)
    integer_nums = list(map(int, nums))
    for number in integer_nums:
        if number % 2 == 0:
            even += number
        elif number % 2 != 0:
            odd += number

    return f'Odd sum = {odd}, Even sum = {even}'


n = input()
print(sum_even_odd_digits(n))

File: JS Front-End/JS Front-End - февруари 2023/Homework - Labs and Exercises/Functions and Statements - Exercises/04. Odd And Even Sum.js
function oddAndEvenSum(num) {
    let stringNum = String(num);
    let oddSum = 0;
    let evenSum = 0;

    for (const n of stringNum) {
        if (Number(n) % 2 === 0) {
            evenSum += Number(n);
        } else {
            oddSum += Number(n);
        }
    }

    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`)
}